R/g_forest.R

```diff
@@ -302,19 +303,18 @@ forest_grob <- function(tbl,
                         name = NULL,
                         gp = NULL,
                         vp = NULL) {
-  stopifnot(
-    !is.null(vline) || is.null(forest_header),
-    is.null(forest_header) || length(forest_header) == 2,
-    is.null(vline) || length(vline) == 1
-  )
-
   nr <- nrow(tbl)
-  stopifnot(
-    is.numeric(x) && length(x) == nr,
-    is.numeric(lower) && length(lower) == nr,
-    is.numeric(upper) && length(upper) == nr,
-    is.null(symbol_size) || length(symbol_size) == nr
-  )
+  if (is.null(vline)) {
+    checkmate::assert_true(is.null(forest_header))
+  } else {
+    checkmate::assert_number(vline)
+    checkmate::assert_character(forest_header, len = 2, null.ok = T)
```

Comment:
```suggestion
    checkmate::assert_character(forest_header, len = 2, null.ok = TRUE)
```

https://www.r-bloggers.com/2012/12/r-tip-avoid-using-t-and-f-as-synonyms-for-true-and-false/

Reply:
that was an error sorry